extensions/ql-vscode/src/cli.ts

```diff
@@ -1138,23 +1138,31 @@ export class CodeQLCliServer implements Disposable {
   /**
    * Gets information about available qlpacks
    * @param additionalPacks A list of directories to search for qlpacks before searching in `searchPath`.
```

Comment:
Thanks for updating the doc comment. Sorry but `seachPath` is also mentioned in this line.